clang/test/CodeGenCXX/x86_64-vaarg.cpp

```diff
@@ -21,3 +21,18 @@ empty empty_record_test(int z, ...) {
   __builtin_va_start(list, z);
   return __builtin_va_arg(list, empty);
 }
+
```

Comment:
Please regenerate the CHECK lines with update_cc_test_checks.py.

Reply:
done